pkg/apis/core/annotation_key_constants.go

```diff
@@ -82,4 +82,22 @@ const (
 	//
 	// Not all cloud providers support this annotation, though AWS & GCE do.
 	AnnotationLoadBalancerSourceRangesKey = "service.beta.kubernetes.io/load-balancer-source-ranges"
+
+	// EndpointsLastChangeTriggerTime is the annotation key, set for endpoints objects, that
+	// represents the timestamp (in milliseconds) of the last change, of some pod or service object,
```

Comment:
Since the unix epoch.
Updated to be more precise.

Reply:
But why not the same RFC 3339 string we use elsewhere?

Reply:
Oh, I was thinking about it in terms of measuring the SLI, storing this as millis here and then in kube-proxy exporting currentMillis - annotationMillis seemed to me like the most straightforward thing. 

But given that annotations are publicly readable / writable it may be indeed better to store is as human-readable RFC 3339 string.

Done.